### PR TITLE
Context for analytics

### PR DESCRIPTION
**Pull request by wolff95**
- Added context and its methods to analytics SDK, configure locale, market, region

### CODE REVIEW

File: packages/bundle/src/initialize.ts [hunk 1]
```diff
@@ -67,10 +67,15 @@ export default async (_config, sentry) => {
 
   __root.config = fromJS(cfg);
   __root.sentry = sentry;
-
+  console.log('CFG', cfg);
```
Comment (pascalachkar) [inline packages/bundle/src/initialize.ts:70]: This is not needed in prod

File: packages/analytics/src/index.ts [hunk 2]
```diff
@@ -84,8 +85,18 @@ const sendEventCreator = ({ events, key, user }: Config) => (
   emitter.emit(event, properties);
 
   const currentUser = user ? user : getUser();
-
-  return api({ key, event, properties, user: currentUser }, endpoint);
+  const currentContext = getContext();
```
Comment (pascalachkar) [inline packages/analytics/src/index.ts:88]: What if there is no context ? where is this handled ?

File: packages/bundle/src/initialize.ts [hunk 3]
```diff
@@ -67,10 +67,15 @@ export default async (_config, sentry) => {
...
   /** Setup analytics */
   __root.analytics = AnalyticsDOM(
-    { platform: cfg.platform, key: cfg.key, events: cfg.analytics || {} },
+    {
+      platform: cfg.platform,
+      key: cfg.key,
+      events: cfg.analytics,
+      context: cfg.context || {},
```
Comment (pascalachkar) [inline packages/bundle/src/initialize.ts:77]: I think here it is better to not send a context at all if it does not exist like this 
```
{
      platform: cfg.platform,
      key: cfg.key,
      events: cfg.analytics,
      ...(cfg.context && { context: cfg.context }),
 }

File: packages/analytics/src/index.ts [hunk 4]
```diff
@@ -84,8 +85,18 @@ const sendEventCreator = ({ events, key, user }: Config) => (
...
+
+  return api(
+    {
+      key,
+      event,
+      properties,
+      user: currentUser,
+      context: currentContext,
```
Comment (pascalachkar) [inline packages/analytics/src/index.ts:96]: Here as well let us not send it if it does not exist

File: packages/bundle/src/initialize.ts (hunk 1 above)
Comment (wolff95) [reply packages/bundle/src/initialize.ts]: Removed

File: packages/bundle/src/initialize.ts (hunk 3 above)
Comment (wolff95) [reply packages/bundle/src/initialize.ts]: Not initialiazed if not configured

File: packages/analytics/src/index.ts (hunk 4 above)
Comment (wolff95) [reply packages/analytics/src/index.ts]: Not sent if null

**Review by pascalachkar — APPROVED** [review]
Approved !!

**Pull request merged**